src/integrationtest/resources/responses/rulesTemplate.json

```diff
@@ -0,0 +1,51 @@
+[
+    {
```

Comment:
Couldn't the template files be reused from the main directory? Do we have to duplicate a lot of resource files for all the tests?

Reply:
I agree with Emelie that the resources should as much as possible to be reused.

Reply:
The template files only exist in back end and so I can't refer to those files. What I could do though is not compare the entire text file and perhaps only compare some other identifier, if at all.